libkbfs/backpressure_disk_limiter_test.go

```diff
@@ -420,7 +420,7 @@ func TestDefaultDoDelayCancel(t *testing.T) {
 	cancel()
 
 	err := defaultDoDelay(ctx, individualTestTimeout)
-	require.Equal(t, ctx.Err(), errors.Cause(err))
+	require.Equal(t, context.Canceled, errors.Cause(err))
```

Comment:
This change isn't required for correctness, right? `defaultDoDelay` should always call `ctx.Done()` in that case.  (Am fine with the change, just making sure I understand.)